src/main/java/Task.java

```diff
@@ -0,0 +1,22 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
+    }
+    public void markAsDone(){
```

Comment:
Should this method be named `setIsDone` since it is a setter for the `isDone` class variable?